src/client/components/BookingForms/AddMeal.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import "./forms.css";
+import { useForm } from "react-hook-form";
+import { joiResolver } from "@hookform/resolvers/joi";
+import Joi from "joi";
+
+const mealsValidSchema = Joi.object({
+  id: Joi.number().integer(),
+  title: Joi.string().required(),
+  description: Joi.string(),
+  location: Joi.string().required(),
+  created_date: Joi.date(),
+  Hosting_time: Joi.date().greater("now").required(),
+  max_number_of_guests: Joi.number().integer().min(10).max(100).required(),
+  price: Joi.number().precision(2).required(),
+});
+
+function AddMeal({ closePopUp }) {
+  const [title, setTitle] = useState("");
```

Comment:
Consider putting these into a single object, your `formState`, else you just have this data clump of related values that are always used together.